metadata-ingestion/src/datahub/ingestion/source/s3/source.py

```diff
@@ -664,7 +706,9 @@ def get_dir_to_process(self, bucket_name: str, folder: str) -> str:
         else:
             return folder
 
-    def s3_browser(self, path_spec: PathSpec) -> Iterable[Tuple[str, datetime, int]]:
+    def s3_browser(
+        self, path_spec: PathSpec, sample_size: int = SAMPLE_SIZE
+    ) -> Iterable[Tuple[str, datetime, int]]:
```

Comment:
Does this ever use the `SAMPLE_SIZE` default now? Just for backwards compatibility? Ideally I think we get rid of this default, so we only have one default -- the one specified in the config field